### PR TITLE
Add list_item element

### PR DESCRIPTION
**Pull request by d2bit**
captura de pantalla 2015-05-07 a las 11 41 50

### CODE REVIEW

**Comment by gponsu** [comment]
:+1

**Comment by maxvlc** [comment]
:+1:

File: app/assets/stylesheets/psique/components/_list-item.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+.list-item-comments {
+  color: $list-item-comments-color;
+  display: inline;
+  font-size: 1rem;
+  margin-left: 1rem;
+  .fa { color: $list-item-comments-icon-color; }
+}
+
+.list-item-detail > * {
```
Comment (Fernan2) [inline app/assets/stylesheets/psique/components/_list-item.scss]: what's this? I've never seen this notation...

**Comment by Fernan2** [comment]
+1

**Comment by scruti** [comment]
:+1:

File: app/assets/stylesheets/psique/components/_list-item.scss (hunk 1 above)
Comment (d2bit) [inline app/assets/stylesheets/psique/components/_list-item.scss]: any element as direct children of `.list-item-detail`

Comment (gponsu) [inline app/assets/stylesheets/psique/components/_list-item.scss]: Not only works in SCSS, is a CSS standard rule :)

Comment (Fernan2) [inline app/assets/stylesheets/psique/components/_list-item.scss]: I didn't know, thanks for the explanation!

**Pull request merged**